plugin.cmake

```diff
@@ -14,6 +14,11 @@
 #  limitations under the License.
 ###############################################################################
 
+# support the "if(TEST ...)" command
+if(POLICY CMP0064)
```

Comment:
If adding custom environment variables is something we'll want to keep doing, that seems like a good approach. Since these changes make for a quick fix for those of us on cmake>=3.4, if that's liable to be more complicated it might be worth doing as a separate PR.